src/main/java/com/aws/greengrass/clientdevices/auth/metrics/ClientDeviceAuthMetrics.java

```diff
@@ -130,6 +136,26 @@ public List<Metric> collectMetrics() {
                 .build();
         metricsList.add(metric);
 
+        metric = Metric.builder()
+                .namespace(NAMESPACE)
+                .name(METRIC_GET_CLIENT_DEVICE_AUTH_TOKEN_SUCCESS)
+                .unit(TelemetryUnit.Count)
+                .aggregation(TelemetryAggregation.Sum)
+                .value(getClientDeviceAuthTokenSuccess.getAndSet(0L))
+                .timestamp(timestamp)
+                .build();
+        metricsList.add(metric);
+
+        metric = Metric.builder()
+                .namespace(NAMESPACE)
+                .name(METRIC_GET_CLIENT_DEVICE_AUTH_TOKEN_FAILURE)
+                .unit(TelemetryUnit.Count)
+                .aggregation(TelemetryAggregation.Sum)
+                .value(getClientDeviceAuthTokenFailure.getAndSet(0L))
+                .timestamp(timestamp)
+                .build();
+        metricsList.add(metric);
+
```

Comment:
Do we want to remove metrics without counts?

Reply:
I can check if a metric has a positive value before building it to begin with